spec/rails_helper.rb

```diff
@@ -5,3 +5,5 @@
 
 ActiveRecord::Migration.maintain_test_schema!
 ActiveRecord::Schema.verbose = false
+
+require 'spec_helper'
```

Comment:
Does this need to be here? Seems like we are creating a circular dependency here with https://github.com/learn-co/railway_ipc_gem/blob/85ea579bd358cda2971adcf4f589a53ea6784f64/spec/spec_helper.rb#L5.